docs/collector.mssql.md

```diff
@@ -220,8 +220,8 @@ Name | Description | Type | Labels
 `windows_mssql_memmgr_target_server_memory_bytes` | Indicates the ideal amount of memory the server can consume | counter | `mssql_instance`
 `windows_mssql_memmgr_total_server_memory_bytes` | Specifies the amount of memory the server has committed using the memory manager | counter | `mssql_instance`
 `windows_mssql_sqlstats_auto_parameterization_attempts` | Number of failed auto-parameterization attempts per second. This should be small. Note that auto-parameterizations are also known as simple parameterizations in later versions of SQL Server | counter | `mssql_instance`
-`windows_mssql_sqlstats_batch_requests` | _Not yet documented_ | counter | `mssql_instance`
-`windows_mssql_sqlstats_failed_auto_parameterization_attempts` | _Not yet documented_ | counter | `mssql_instance`
+`windows_mssql_sqlstats_batch_requests` | Number of SQL batch requests received by server | counter | `mssql_instance`
+`windows_mssql_sqlstats_failed_auto_parameterization_attempts` | Number of failed auto-parameterizations | counter | `mssql_instance`
```

Comment:
Are these totals or current numbers? If totals they'll need the `_total` suffix, if not we'll need to change them to gauges.